recipes/pyproject_hooks/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pyproject_hooks" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyproject_hooks-{{ version }}.tar.gz
+  sha256: f271b298b97f5955d53fb12b72c1fb1948c22c1a6b70b315c54cedaca0264ef5
+
+build:
+  skip: true  # [py<37]
```

Since this is pure Python it would be much better to make this noarch and just unconditionally install tomli.
```suggestion
  noarch: python
```